### PR TITLE
Document bad interactions with `br_netfilter`

### PR DESCRIPTION
**Pull request by simondeziel**

### CODE REVIEW

File: doc/howto/network_bridge_firewall.md [hunk 1]
```diff
@@ -177,3 +177,20 @@ Allow egress network traffic flows
...
   ```
+
+```{note}
+The Docker snap is also known to load the `br_netfilter` module, see {ref}`network-lxd-br-netfilter`.
+```
+
+(network-lxd-br-netfilter)=
+## Prevent connectivity issues with LXD and bridge filtering
+
+When using LXD bridges, the `br_netfilter` modules must not be loaded. When that module is loaded, even internal bridge traffic becomes subjects to firewall evaluation. This means that two instances connected to the same bridge would be unable to communicate with multicast as the traffic would be blocked at the firewall level.
```
[the remark this answers is not in the document]
Comment (tomponline) [reply doc/howto/network_bridge_firewall.md:188]: So "must not" is too strong in my view

[the remark this answers is not in the document]
Comment (simondeziel) [reply doc/howto/network_bridge_firewall.md:188]: Oh, if that's fixed it must not be in `5.21/stable` which is the version I believe @minaelee was using to run her test microcloud.

[the remark this answers is not in the document]
Comment (tomponline) [reply doc/howto/network_bridge_firewall.md:188]: yeah its not in there yet, try 5.21/edge

[the remark this answers is not in the document]
Comment (simondeziel) [reply doc/howto/network_bridge_firewall.md:188]: @minaelee would you be so kind and test updating to `5.21/edge` and see if your MicroCloud VMs behave normally even when `br_netfilter` is loaded and the Docker snap is enabled? Thanks

**Comment by tomponline** [comment]
@simondeziel @minaelee can we close this until the testing is done, as trying to clear out some of the long-lived PRs?

**Comment by simondeziel** [comment]
@tomponline Minae and I have a quick meet scheduled for later today so this will be dealt with today.

**Comment by simondeziel** [comment]
For some reasons, we were unsuccessful in reproducing the initial issue so let's close this PR. @tomponline I'm assuming it's a matter of time for LXD 5.0 to get the firewall update as well, right?

**Comment by tomponline** [comment]
> I'm assuming it's a matter of time for LXD 5.0 to get the firewall update as well, right?

Probably not, we might need to update the lxd-ci tests, as that isn't a "critical bug" and might cause regressions, so am not keen to backport that to 5.0.

**Comment by simondeziel** [comment]
> > I'm assuming it's a matter of time for LXD 5.0 to get the firewall update as well, right?
> 
> Probably not, we might need to update the lxd-ci tests, as that isn't a "critical bug" and might cause regressions, so am not keen to backport that to 5.0.

In lxd-ci, that `br_netfilter` is unloaded unconditionally to better match a "real system" so it shouldn't be an issue.

**Pull request closed without merging**